.gitignore

```diff
@@ -5,6 +5,7 @@ coverage.xml
 .connect-extension-xvs_dev.env
 node_modules
 ui/tests/coverage
+cbc_pg_data
```

Comment:
Didn't get why we need this ignore here, can you please put more details?

Reply:
This need to be ignored because, this directory will contains pg_data for CBC Database included in docker-compose.yml